fscache_test.go

```diff
@@ -263,3 +263,31 @@ func TestConcurrent(t *testing.T) {
 		}
 	})
 }
+
+func TestSize(t *testing.T) {
+	testCaches(t, func(c Cache) {
+		defer c.Clean()
+
+		l := c.Size("dankmemes")
+		if l != 0 {
```

Comment:
After returning the error, now you can test that there is an error, instead of size 0.